### DISCUSSION TITLE
Is it possible to make Laragon not detect a folder as a project?

### DISCUSSION THREAD

**Discussion by Lexian-droid** [leokhoa/laragon#550] [General]
I'm new to Laragon, just switched today, actually. I was using MAMP prior to it. and I've changed the default root directory to a page I made, however I realized it detects all of the folders as a project file.

Is there any way to prevent this from happening?

**Comment by Pen-y-Fan** [comment]
The way Laragon handles projects is different to MAMP. All folders in the laragon/**www** directory are automatically detected as a project. See [Directory Structure](https://laragon.org/docs/directory-structure.html)

Change the project route back to the **www** directory, copy/move your projects to that directory and they will be automatically detected as projects.

**Reply by Lexian-droid** [reply]
Hello, I did figure that it was probably not possible, since it seemed to just detect folders, rather then look for anything specific.
To clarify, I used the regular version of MAMP - Not MAMP Pro. This type of ecosystem is actually quite foreign to me.

However that being said, I do appreciate your response, thank you. I did move the directory for laragon on purpose, I just didn't anticipate that the assets would also be detected, and therefore hosted on a server.

**Reply by LebToki** [reply]
[Shall I recommend my own Index Replacement Project ? Do have a solution there for you next to tons of other features that would surely add value](https://github.com/LebToki/Index-for-Laragon)

**Reply by Lexian-droid** [reply]
Unfortunately, that's only an index replacement, and I usually prefer my own index replacement from third parties. My issue would be more in the realm of server settings, however i've given up on the idea since then.

**Reply by LebToki** [reply]
You see laragon was built on the basis that regardless of the stack version you are running you have the full flexibility to update everything around it so yes I understand your point but then again here's one way to work around your requirement for now at least.

**Comment by MichaelSentry** [comment]
You can add an IgnoreList setting to the laragon/usr/laragon.ini file to prevent specific directories from being detected as projects.

```
[preferences]

IgnoreList=cms|editor|frameworks|fortified|forums|git
```

**Reply by Lexian-droid** [reply]
Thank you!

**Comment by Lexian-droid** [comment]
> You can add an IgnoreList setting to the laragon/usr/laragon.ini file to prevent specific directories from being detected as projects.
> 
> ```
> [preferences]
> 
> IgnoreList=cms|editor|frameworks|fortified|forums|git
> ```
